packages/core/src/logger.js

```diff
@@ -5,17 +5,43 @@
 
 'use strict';
 
+/**
+ * @type {*}
+ */
 let parentLogger = null;
+/**
+ * @type {*}
+ */
 const registry = {};
 
+/**
+ * @typedef {Object} GenericLogger
+ * @property {(...args: *) => void} debug
+ * @property {(...args: *) => void} info
+ * @property {(...args: *) => void} warn
+ * @property {(...args: *) => void} error
+ */
+
+/**
+ * @type {GenericLogger}
+ */
 const consoleLogger = {
   /* eslint-disable no-console */
   debug: console.log,
   info: console.log,
   warn: console.warn,
   error: console.error
+  /* eslint-enable no-console */
```

Comment:
I think I got used to "close" these ESLint comments, as it could potentially hide unwanted/unexpected linting behavior in the future.
It doesn't seem to be the case here, so I will remove it...

Reply:
Ah, I misread this. I didn't realize this is `enable`, I thought you are repeating the `disable` here.

I think eslint-disable comments are scoped to the enclosing block but actually this is not a block but just an object literal. I just tested this quickly, the `enable` is _required_ here, you are right. Please leave it in.